src/SFA.DAS.ApprenticeFeedback.Api.IntegrationTests/SFA.DAS.ApprenticeFeedback.Api.IntegrationTests.csproj

```diff
@@ -38,7 +38,7 @@
     <None Update="connectionStrings.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="connectionStrings.local.json">
+    <None Update="connectionStrings.Local.json">
```

Comment:
@ChrisJWoodcock is this change necessary?

Reply:
Not totally but the file was not being grouped correctly in the solution explorer